src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -58,7 +59,19 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
               artwork={artwork}
             />
             {hasEnded ? (
-              <BiddingClosedMessage />
+              <BiddingClosedMessage>
+                {!!shouldShowCreateAlertSection && (
```

Comment:
use `isCreateAlertButtonForArtworkEnabled` instead of `shouldShowCreateAlertSection` here